bundletool/bundletool.go

```diff
@@ -5,44 +5,41 @@ import (
 	"net/http"
 	"path/filepath"
 
-	"github.com/bitrise-io/go-utils/command"
 	"github.com/bitrise-io/go-utils/pathutil"
 )
 
-// Runner ...
-type Runner struct {
-	localPath string
-}
+// BundleTool ...
+type BundleTool string
```

Comment:
bundletool.BundleTool is not conventional. this type holds the path of the tool, lets name it `Path` then.
`bundletool.Path = "/path/to/the/bundle/tool"` makes real sense for me.